contrib/debian/bitcoind.postinst

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# setup bitcoin account, homedir etc
+
+set -e
+
+BCUSER="bitcoin"
+BCHOME="/var/lib/bitcoin"
+
+if [ "$1" = "configure" ]; then
+
+	# Add bitcoin user/group - this will gracefully abort if the user already exists.
+	# A homedir is never created.
```

Comment:
Why not create the directory? If its empty it will be deleted on uninstall, but if ther user ever tries to use the service things will simply fail to start, no?

Reply:
The reason for this approach is that, if for whatever reason the user has changed ownership or permissions on the `/var/lib/bitcoin` directory, we don't want to mess with that on operations like reconfigure, update etc.

`adduser` does not let me specify custom homedir permissions, so I'm using `--no-create-home` to work around that. A missing home directory becomes a stable indicator that this is a fresh install, making it safe for me to create the directory and set the restrictive default permissions.

Reply:
Oh, I misread the script, sorry.